### PR TITLE
docs(vrl): add documentation for `parse_etld` function

### PR DESCRIPTION
**Pull request by esensar**
Related: https://github.com/vectordotdev/vrl/pull/669

### CODE REVIEW

**Review by hestonhoffman — APPROVED** [review]

**Review by jszwedko — APPROVED** [review]
Thanks @esensar !

**Review by pront — APPROVED** [review]
Thanks! We will have to wait for the next VRL release update before we can get this merged.

**Pull request merged**
